src/MockClassifier.Api/Services/NaturalLanguageService.cs

```diff
@@ -0,0 +1,25 @@
+﻿using MockClassifier.Api.Models;
+using MockClassifier.Api.Interfaces;
+
+namespace MockClassifier.Api.Services
+{
+    public class NaturalLanguageService : IClassifierService
+    {
+
+        public string[] Classify(string messageBody)
+        {
+            var token = "";
+            switch (messageBody)
```

Comment:
nit: I may be incorrect here, but would a switch expression make this a little more succinct? https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/switch-expression

Reply:
resolved in latest changes